src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioWorkspaceHost.cs

```diff
@@ -34,5 +34,6 @@ internal interface IVisualStudioWorkspaceHost
         void OnAdditionalDocumentOpened(DocumentId documentId, ITextBuffer textBuffer, bool isCurrentContext);
         void OnAdditionalDocumentClosed(DocumentId documentId, ITextBuffer textBuffer, TextLoader loader);
         void OnAdditionalDocumentTextUpdatedOnDisk(DocumentId id);
+        void OnIntellisenseBuildResult(ProjectId projectId, bool succeed);
```

Comment:
I would prefer a separate interface so that people can continue to install newer Roslyn builds without having to disable CodeLens.